### ISSUE TITLE
IDE0060 incorrectly reported for record parameters decorated with attributes

### ISSUE THREAD

**Issue by WilStead** [dotnet/roslyn#47143]
**Version Used**: Version 16.8.0 Preview 2.0

**Steps to Reproduce**:
(Taken directly from the [ASP.NET Core .NET 5 Preview 8 blog post](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-5-preview-8/))
```c#
public record Person([Required] string Name, [Range(0, 150)] int Age);
```

**Expected Behavior**: No warnings.

**Actual Behavior**: **IDE0060** reported for both parameters (properties?) in the short syntax record definition.

**Comment by RikkiGibson** [comment]
Perhaps a duplicate of #47142

**Comment by jinujoseph** [comment]
cc @CyrusNajmabadi

**Comment by sharwell** [comment]
Duplicate of #47142
